src/App.tsx

```diff
@@ -57,6 +57,8 @@ const App: FC = () => {
       dispatch(getGroupsThunk()),
       dispatch(getPlanningClassesThunk()),
       dispatch(getPlanningLocationsThunk()),
+      dispatch(getGroupsThunk()),
```

Comment:
getGroupsThunk is now called twice, it should be named to getPlanningGroupsThunk and only called ones. The action should be named to 'groups/getAllPlanning'.